.pre-commit-config.yaml

```
@@ -360,3 +360,8 @@ repos:
         entry: "./scripts/ci/pre_commit_bat_tests.sh"
         files: ^breeze$|^breeze-complete$|\.sh$|\.bash$
         pass_filenames: false
+      - id: stylelint
+        name: stylelint
+        entry: "stylelint"
+        language: node
+        files: ^airflow/www/.*\.(css|scss|sass)$
```

```suggestion
        files: ^airflow/www/.*\.(css|scss|sass)$
        additional_dependencies: ["stylelint@13.3.0"]
```
This should also work. Please look at the example:  https://github.com/pre-commit/mirrors-eslint/blob/master/.pre-commit-hooks.yaml

You can also use: https://github.com/awebdeveloper/pre-commit-stylelint

I don't know which solution is better, but downloading a large image with the system for one simple application can be expensive. I suspect that there is no one common base image for all pre-commit hooks.

---

Should we also add `stylelint-config-standard": "^20.0.0`?

---

ah, yes, that needs to be added too. Thanks!